src/main/drivers/pwm_output_dshot_shared.c

```diff
@@ -243,6 +243,8 @@ FAST_CODE_NOINLINE bool pwmStartDshotMotorUpdate(void)
         }
         pwmDshotSetDirectionOutput(&dmaMotors[i]);
     }
+
+    dshotTelemetryState.combinedDataNeedsUpdate = true;
```

Comment:
```suggestion
    updateDshotAverageRpm();
```

Reply:
Reworked

Reply:
Is there a reason you don't want to update the average Rpm in dshot.c and are hiding the implementation in `dshot_bitbang.c` and `pwm_output_dshot_shared.c`?